### PR TITLE
SW-2175 Allow accession quantity to be editable in every state

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#2466** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2466?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/strings/csv/en.csv [hunk 1]
```diff
@@ -407,7 +407,7 @@ EDIT_PLANTING_SITE,Edit Planting Site,
 EDIT_PROJECT,Edit Project,
 EDIT_PROJECT_NAME_CONFIRMATION,You are editing a project name that is currently being used by {0} in Terraware.,
 EDIT_QUANTITY,Edit Quantity,
-EDIT_QUANTITY_DISABLED,"Currently, you may only input a quantity when the status is ""Drying"" or “In Storage.”",
+EDIT_QUANTITY_DISABLED,"Quantity may not be editted when the status is ""Used Up"".",
```
Comment (karthikbtf) [inline src/strings/csv/en.csv:410]: `edited`?

**Review by karthikbtf — APPROVED** [review]
looks good, one comment

File: src/strings/csv/en.csv (hunk 1 above)
Comment (karthikbtf) [reply src/strings/csv/en.csv:410]: @tommylau523 check the spelling here?

**Pull request merged**